src/app/components/Curation/HierarchicalGrid/index.tsx

```diff
@@ -140,9 +140,13 @@ const HiearchicalGrid = ({
                 <Promo.Body className="promo-paragraph" css={styles.body}>
                   {promo.description}
                 </Promo.Body>
-                <Promo.Timestamp className="promo-timestamp">
-                  {promo.lastPublished}
-                </Promo.Timestamp>
+                {!isLive ? (
+                  <Promo.Timestamp className="promo-timestamp">
+                    {promo.lastPublished}
+                  </Promo.Timestamp>
+                ) : (
+                  <React.Fragment />
```

Comment:
Instead of using an empty `div`

Reply:
Have now swapped for an empty React Fragment instead 😄